lib/experimental/block-editor-settings-mobile.php

```diff
@@ -33,7 +67,64 @@ function gutenberg_get_block_editor_settings_mobile( $settings ) {
 		$settings['__experimentalEnableListBlockV2'] = true;
 	}
 
-	return $settings;
+	return keep_supported_block_editor_settings_mobile(
+		$settings,
+		array(
+			'alignWide'                        => true,
+			'allowedBlockTypes'                => true,
+			'allowedMimeTypes'                 => true,
+			'defaultEditorStyles'              => true,
+			'blockCategories'                  => true,
+			'isRTL'                            => true,
+			'imageDefaultSize'                 => true,
+			'imageDimensions'                  => true,
+			'imageEditing'                     => true,
+			'imageSizes'                       => true,
+			'maxUploadFileSize'                => true,
+			'__unstableGalleryWithImageBlocks' => true,
+			'disableCustomColors'              => true,
+			'disableCustomFontSizes'           => true,
+			'disableCustomGradients'           => true,
+			'disableLayoutStyles'              => true,
+			'enableCustomLineHeight'           => true,
+			'enableCustomSpacing'              => true,
+			'enableCustomUnits'                => true,
+			'colors'                           => true,
+			'fontSizes'                        => true,
+			'__experimentalFeatures'           => array(
+				'appearanceTools'               => true,
+				'useRootPaddingAwareAlignments' => true,
+				'border'                        => true,
+				'color'                         => true,
+				'shadow'                        => true,
+				'spacing'                       => true,
+				'typography'                    => array(
+					'dropCap'        => true,
+					'fontSizes'      => true,
+					'fontStyle'      => true,
+					'fontWeight'     => true,
+					'letterSpacing'  => true,
+					'textColumns'    => true,
+					'textDecoration' => true,
+					'textTransform'  => true,
+					'writingMode'    => true,
+				),
+				'blocks'                        => true,
+				'background'                    => true,
+				'dimensions'                    => true,
+				'position'                      => true,
+			),
+			'gradients'                        => true,
+			'disableCustomSpacingSizes'        => true,
+			'spacingSizes'                     => true,
+			'__unstableIsBlockBasedTheme'      => true,
+			'localAutosaveInterval'            => true,
+			'__experimentalDiscussionSettings' => true,
+			'__experimentalDashboardLink'      => true,
+			'__experimentalEnableQuoteBlockV2' => true,
+			'__experimentalEnableListBlockV2'  => true,
+		)
```

Comment:
I'll preface this thought with the fact that it is has been a very long time since I last wrote PHP. I'm less familiar with its current syntax or paradigms. 

Would it make sense to hoist this array value and assign it to a variable/constant? I wonder if it would make the code a bit easier to comprehend? WDYT?

Reply:
After reviewing the response from the API via `curl`, the values in this array appears to match the network response. I have not tested functionality in the editor, but this list makes sense to me.

Reply:
> it is has been a very long time since I last wrote PHP. I'm less familiar with its current syntax or paradigms.

I'm in the same boat 😅 

> Would it make sense to hoist this array value and assign it to a variable/constant?

That does make sense to me. 👍

Reply:
Resolved w/ https://github.com/WordPress/gutenberg/pull/56424/commits/7f3d7052ebc2bea9847708041bd9a4d6118cb00d